src/zenml/materializers/pandas_materializer.py

```diff
@@ -11,182 +11,27 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
-"""Materializer for Pandas."""
+"""Implementation of ZenML's pandas materializer."""
```

Comment:
Doesn't feel accurate, since you just import it here.

Reply:
Makes sense. Changed it accordingly.